docs/markdown/snippets/rustc-ndebug.md

```diff
@@ -0,0 +1,5 @@
+## Rust now supports the b_ndebug option
+
+Which controls the `debug_assert!()` macro. This macro is roughly equivalent to
+C's `assert()`, as it can be toggled with command line options, unlike Rust's
```

Comment:
Perhaps add the man page section that assert belongs to? Seems to be 3. Might help make this a little clearer.

In addition `assert` in Rust is a macro, right? Would it be more correct to add the `!`?